recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ requirements:
     - mumps-mpi
     - suitesparse
     - hdf5
+    - fftw * mpi_{{ mpi }}_*  # [mpi != 'nompi']
```

Comment:
```suggestion
    - fftw {{ fftw }} mpi_{{ mpi }}_*  # [mpi != 'nompi']
```

Reply:
thanks!
should this change be done for `host` and `run` or only for `host`?

Reply:
Only needed for host, but doing it for both is fine too.

Reply:
```suggestion
    - fftw {{ fftw }}.* mpi_{{ mpi }}_*  # [mpi != 'nompi']
```

Reply:
does not work